prime-router/docs/design/design/RS-Extention-Registry.yml

```diff
@@ -0,0 +1,5930 @@
+              valueType: String
+              hl7Use: HD.2
+            - id: universal-id-type
+              url: "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-type"
+              valueType: String
+              hl7Use: HD.3
+    SFT:
+      Example:
+        {
+          "fullUrl": "MessageHeader/827ccb0e-ea8a-306c-8c34-a16891f84e7b",
+          "resource": {
+            "resourceType": "MessageHeader",
+            "id": "827ccb0e-ea8a-306c-8c34-a16891f84e7b",
+            "extension": [
+              {
+                "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/encoding-characters",
+                "valueString": "^~\\&#"
+              },
+              {
+                "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/character-set",
+                "valueString": "UNICODE UTF-8"
+              }
+            ],
+            "source": {
+              "extension": [
+                {
+                  "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/software-binary-id",
+                  "valueString": "Binary ID unknown"
+                },
+                {
+                  "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/source-software-description",
+                  "valueString": "description"
+                },
+                {
+                  "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/software-install-date",
+                  "valueString": "20220411"
+                },
+                {
+                  "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/software-vendor-org",
+                  "valueReference": {
+                    "reference": "Organization/1707483834125725000.05476f90-2bd7-413d-8ecb-e705778cff00"
+                  }
+                }
+              ]
+            }
+          }
+        }
+      Extension:
+        - id: software-vendor-org
+          url: "https://reportstream.cdc.gov/fhir/StructureDefinition/software-vendor-org"
+          valueType: Reference
+          hl7Use: SFT.1
+        - id: software-binary-id
+          url: "https://reportstream.cdc.gov/fhir/StructureDefinition/software-binary-id"
+          valueType: String
+          hl7Use: SFT.4
+        - id: source-software-description
+          url: "https://reportstream.cdc.gov/fhir/StructureDefinition/source-software-description"
+          valueType: String
+          hl7Use: SFT.5
+        - id: software-install-date
+          url: "https://reportstream.cdc.gov/fhir/StructureDefinition/software-install-date"
+          valueType: String
+          hl7Use: SFT.6
+    HD:
+      Example:
+        {
+          "fullUrl": "MessageHeader/827ccb0e-ea8a-306c-8c34-a16891f84e7b",
+          "resource": {
+            "resourceType": "MessageHeader",
+            "id": "827ccb0e-ea8a-306c-8c34-a16891f84e7b",
+            "extension": [
```

Comment:
Nit: maybe delete the bit here that ins't from an `HD` or comment where the HD is?

Reply:
I left MessageHeader resource in formation in the drive home that this is only on the MessageHeader resource. The individual HD fields are marked in the Extension section. Think that's sufficient?